src/hardware/input/intel8042.cpp

```diff
@@ -902,6 +915,10 @@ static void execute_command(const Command command, const uint8_t param)
 		// firmware allow everything? writing bits 4,5 and 6
 		// should be safe in real implementation, how about here?
 		sanitize_config_byte();
+		// Make sure only the needed IRQs are activated
+		PIC_DeActivateIRQ(get_irq_mouse());
+		PIC_DeActivateIRQ(get_irq_keyboard());
+		activate_irqs_if_needed();
```

Comment:
Can't deactivating both first then activating only what's needed can cause unwanted side effects? Wouldn't it be better/safer to not deactivate & reactivate what's already active? But maybe that would be worse—I'm just wondering 😄